ateam_ui/src/src/overlay.ts

```diff
@@ -34,6 +36,49 @@ export class Overlay {
         if (this.lifetime) {
             this.lifetime_end = Date.now() + this.lifetime;
         }
+        let frag_src = `
+            vec4 jet (float x) {
+                const float e0 = 0.0;
+                const vec4 v0 = vec4(0,0,0.5137254901960784,1);
+                const float e1 = 0.125;
+                const vec4 v1 = vec4(0,0.23529411764705882,0.6666666666666666,1);
+                const float e2 = 0.375;
+                const vec4 v2 = vec4(0.0196078431372549,1,1,1);
+                const float e3 = 0.625;
+                const vec4 v3 = vec4(1,1,0,1);
+                const float e4 = 0.875;
+                const vec4 v4 = vec4(0.9803921568627451,0,0,1);
+                const float e5 = 1.0;
+                const vec4 v5 = vec4(0.5019607843137255,0,0,1);
```

Comment:
Where are all these constants coming from? Should we have them be some sort of config/variable?

Reply:
You know how I said it was a nightmare trying to get different Cmaps and interpolation schemes from that one glsify package to import correctly? This is the hard coded default from that. Pretty sure we talked about this.

Reply:
We can separate them into a file if you want and just read that file into the string later. Notice it's a variable number of stops. We would need to write what glsify does for auto generating these or just get it working